packages/cw-umee-types/src/query_metoken.rs

```diff
@@ -0,0 +1,124 @@
+use cosmwasm_std::{Coin, Decimal};
+use schemars::JsonSchema;
+use serde::{Deserialize, Serialize};
+
+#[derive(Serialize, Deserialize, Clone, PartialEq, JsonSchema, Debug)]
+#[serde(rename_all = "snake_case")]
+pub enum UmeeQueryMeToken {
+  MetokenParameters(MetokenParametersParams),
+  MetokenIndexes(MetokenIndexesParams),
+  MetokenSwapfee(MetokenSwapfeeParams),
+  MetokenRedeemfee(MetokenRedeemfeeParams),
+  MetokenIndexbalances(MetokenIndexbalancesParams),
+  MetokenIndexPrice(MetokenIndexPriceParams),
+}
+
+#[derive(Serialize, Deserialize, Clone, PartialEq, JsonSchema, Debug)]
+pub struct MetokenParametersParams {}
+
+#[derive(Serialize, Deserialize, Clone, PartialEq, JsonSchema, Debug)]
+pub struct MetokenParametersResponse {
+  pub params: MetokenParameters,
+}
+
+#[derive(Serialize, Deserialize, Clone, PartialEq, JsonSchema, Debug)]
+pub struct MetokenParameters {
+  pub rebalancing_frequency: i64,
+  pub claiming_frequency: i64,
+}
+
+#[derive(Serialize, Deserialize, Clone, PartialEq, JsonSchema, Debug)]
+pub struct MetokenIndexesParams {
+  pub metoken_denom: String,
```

Comment:
In the chain side this has two json naming 
https://github.com/umee-network/umee/blob/main/x/metoken/query.pb.go#L111-L112
type QueryIndexes struct {
	MetokenDenom string `protobuf:"bytes,1,opt,name=metoken_denom,json=metokenDenom,proto3" json:"metoken_denom,omitempty"`
}

We should specify on the proto probably to avoid confusion

Reply:
I think no need for specify the proto json name because we are parsing json messages on handler side